### ISSUE TITLE
Loss

### ISSUE THREAD

**Issue by hdmGOAT** [hdmGOAT/trustformer#7]
- [ ] Log-softmax
- [ ] Cross-entropy loss
- [ ] Masking
